eventlog/el/main.go

```diff
@@ -347,3 +411,327 @@ func search(c *cli.Context) error {
 	}
 	return nil
 }
+
+func login(c *cli.Context) error {
```

Comment:
we should have an integration test for these (but it might be difficult because it relies on external servers, can we mock a server?)

Reply:
Filed issue on this: #1520